index.html

```diff
@@ -73,7 +73,7 @@ <h1>TONIC</h1>
             <li><h5>css</h5></li>
             <li><h5>javascript</h5></li>
          </ul>
-         <button class="btn">See Project</button>
+         <button type="button">See Project</button>
         </div>
 
```

Comment:
- [ ] Please implement hovering for the buttons so they match the design requirements.